win32/devdoc/threadpool_win32.md

```diff
@@ -265,21 +265,21 @@ MOCKABLE_FUNCTION(, THREADPOOL_WORK_ITEM_HANDLE, threadpool_create_work_item, TH
 
 **SRS_THREADPOOL_WIN32_05_005: [** If `work_function` is `NULL`, `threadpool_create_work_item` shall fail and return a `NULL` value. **]**
 
-**SRS_THREADPOOL_WIN32_05_006: [** Otherwise `threadpool_create_work_item` shall allocate a context `work_item_context` of type `THREADPOOL_WORK_ITEM_HANDLE` where `work_function`, `work_function_context`, and `ptp_work` shall be saved. **]**
+**SRS_THREADPOOL_WIN32_05_006: [** Otherwise `threadpool_create_work_item` shall allocate a context `work_item_context` of type `THANDLE(THREADPOOL_WORK_ITEM)` where `work_function`, `work_function_context`, and `ptp_work` shall be saved. **]**
```

Comment:
>a context `work_item_context` of type `THANDLE(THREADPOOL_WORK_ITEM) [](http://example.com/codeflow?start=90&length=68)

I think we should say "shall allocate a THANDLE(THREADPOOL_WORK_ITEM) with the dispose function being threadpool_dispose_work_item

Reply:
Done